packages/mdc-menu-surface/_mixins.scss

```diff
@@ -95,6 +95,8 @@
   @include mdc-feature-targets($feat-structure) {
     display: none;
     position: absolute;
+    top: 0;
```

Comment:
The positions may change based on where the menu surface is anchored and the orientation of menu surface based on viewport size. So I think, it is not safe to hard code the values only for `top` & `left`.

Reply:
Oh I see maybe I need to get position of menu before applying className "--animating-open".

Reply:
I executed autoPosition_ before applying class.
And I moved dimensions setter into autoautoPosition_.
Check my code again please :)

Reply:
Thanks for the changes! Somehow I'm not able to reproduce the original issue on master branch.

Here is the demo:
https://dist-qw5tsda9o.now.sh/

But I'm still able to reproduce this on catalog site with latest version. I'm curious, why it calculates the position correctly outside the catalog.

Reply:
@abhiomkar 
I guess this issue is fired when there is inner scroll.
Because select's menu surface appends to body.

![스크린샷 2019-04-17 오전 2 46 15](https://user-images.githubusercontent.com/20078201/56232240-3e301480-60bb-11e9-8ac1-ff86e3901fad.png)
And catalog website has inner scroll.

Reply:
@abhiomkar 
Can you check this please?